client/executor/wasmtime/src/runtime.rs

```diff
@@ -368,7 +373,7 @@ fn common_config(semantics: &Semantics) -> std::result::Result<wasmtime::Config,
 
 		let mut pooling_config = wasmtime::PoolingAllocationConfig::default();
 		pooling_config
-			.strategy(wasmtime::PoolingAllocationStrategy::ReuseAffinity)
+			.max_unused_warm_slots(4)
```

Comment:
out of curiosity: why 4?

Reply:
It's a low enough number that's bigger than `1`, less than the default of `100` and seems to be enough for us. :P The docs for this [are here](https://docs.rs/wasmtime/latest/wasmtime/struct.PoolingAllocationConfig.html#method.max_unused_warm_slots) if you're interested. They're a little confusing, but if I'm reading them right then the value doesn't really matter as we instantiate a separate instance of `Engine` for each module, so there shouldn't be any other warm slots for the newly instantiated instance to hijack, so essentially this is just a "just-in-case" option.

Reply:
would be good to add this as a comment